tests/test_estimators/test_ensemble_model.py

```diff
@@ -0,0 +1,33 @@
+from sklego.ensemble_model import EnsembleModel
+from sklearn import datasets
+from sklearn.model_selection import train_test_split
+
+# Data
+iris = datasets.load_iris()
+X = iris.data[:, :2]  # we only take the first two features.
+y = iris.target
+X_train, X_test, y_train, y_test = train_test_split(X, y)
+
+# Model
+model = EnsembleModel(X_test = X_test, y_test = y_test)
+model.fit(X_train, y_train)
+
+def test_alpha():
+    assert model.alpha >= 0
+    assert model.alpha <= 1
+
+
+def test_predict_proba():
```

Comment:
in this case there is a module in the testing framework where shapes and other common things are tested for **all** estimators. might be better to put this test there.